app/javascript/components/Results/HealthTopics/HealthTopics.tsx

```diff
@@ -26,7 +26,7 @@ export const HealthTopics = ({ description, title, url, relatedTopics=[], studie
           <Grid col={true}>
             <GridContainer className='health-topic-title'>
               <a href={url}>{parse(title)}</a>
-              <a href={i18n.t('searches.medTopic.homepageUrl')} className={`logo-${i18n.locale}`}>
+              <a href={i18n.t('searches.medTopic.homepageUrl')} className={`logo-${i18n.locale}`} aria-label="MedlinePlus">
```

Comment:
@yogesh27 Can we use the translated label for `aria-label` here?  Assistive tech users should have equal access to localized labels.